src/Components/Users/UserAdd.tsx

```diff
@@ -631,15 +632,26 @@ export const UserAdd = (props: UserProps) => {
                 showAll={false}
               />
             </div>
-            <SelectFormField
+
+            {/* <SelectFormField
```

Comment:
You can remove this instead of commenting it

Reply:
I was checking the everything was working properly I will remove and make new commit.